src/constants.ts

```diff
@@ -61,7 +61,7 @@ export const API_V2_SCANNER_ENGINE_ENDPOINT = '/analysis/engine';
 export const API_OLD_VERSION_ENDPOINT = '/api/server/version';
 
 export const SCANNER_CLI_DEFAULT_BIN_NAME = 'sonar-scanner';
-export const SCANNER_CLI_VERSION = '5.0.1.3006';
+export const SCANNER_CLI_VERSION = '6.0.0.4432';
```

Comment:
CLI version 6 does the exact same thing as the npm scanner. I'm trying to understand when we deprecate the CLI workflow versus support two almost identical paths to the scanner-engine 🤔

Reply:
Using the CLI is necessary to bootstrap SQ < 10.6. So you can't really deprecate/remove this path until the new LTS is out.
Updating the CLI to 6.0 is useful even for SQ < 10.6, as we made several improvements in the logging, SSL configuration, proxy support, ... that could partly be useful with older SQ.

Reply:
Thanks for the input @henryju. We discussed on slack and we agreed there is a big point to deprecate the fallback strategy in the next major release (v5). The docs and the logs of the new release should be explicit when the endpoint API does not support the new bootstrapping and point to npm scanner v4. This will simplify drastically the codebase. But I agree we should wait until 10.6. Also, we could keep updating v4 with new scanner cli versions independently.

Reply:
I can't seem to find a right place to ask this, but Is the new sonar cli scanner only supporting server URLs with SSL (http**s** in URL is mandatory)? 

Full questiion posted here https://community.sonarsource.com/t/sonar-scanner-npm-4-2-0-typeerror-invalid-url/120189

Best regards.

Reply:
Nevermind, the problem was trailing whitespace, it's working now, sorry for the noise.